docs/source/subscription.rst

```diff
@@ -0,0 +1,106 @@
+Subscribe to RedHat Subscription Management
+===========================================
+
+The section 'RH-subscription' can be used to register the system to RHSM,
+attach to a pool, enable repos, disable repos, and unregister from RHSM.
+
+
+**Step 1:**
+
+Create an empty file and give it any arbitrary name. For the purpose of this
```

Comment:
Can you remove that create an empty file ... sentence. And keep it like:

Crate a configuration file, which will look like the one show in the example below.